### PR TITLE
Update qset to match stellarbeat

### PR DESCRIPTION
**Pull request by mollykarcher**
See https://github.com/stellar/kube/pull/2089

### CODE REVIEW

File: charts/horizon/templates/_core-config.tpl
Comment (sreuland) [inline charts/horizon/templates/_core-config.tpl:1]: I tested this locally, I think output looks correct:

```
$ helm template my-horizon ./charts/horizon --values https://raw.githubusercontent.com/stellar/helm-charts/main/charts/horizon/values.yaml --set global.network="pubnet"
---
# Source: horizon/templates/horizon-core-cm.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: my-horizon-core
  namespace: default
  labels:
    app: my-horizon-core
    chart: horizon-0.0.7
    release: my-horizon
    heritage: Helm
data:
  stellar-core.cfg: |
    PUBLIC_HTTP_PORT=true
    UNSAFE_QUORUM=true
    ENABLE_PULL_MODE=true
    INVARIANT_CHECKS=["AccountSubEntriesCountIsValid", "ConservationOfLumens", "ConstantProductInvariant", "LedgerEntryIsValid", "LiabilitiesMatchOffers", "SponsorshipCountIsValid"]
    EXPERIMENTAL_BUCKETLIST_DB=true
    EXPERIMENTAL_BUCKETLIST_DB_INDEX_PAGE_SIZE_EXPONENT=12
    [[HOME_DOMAINS]]
    HOME_DOMAIN="www.stellar.org"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="stellar.blockdaemon.com"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="coinqvest.com"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="satoshipay.io"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="publicnode.org"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="lobstr.co"
    QUALITY="HIGH"
    [[VALIDATORS]]
    ADDRESS="core-live-a.stellar.org"
    HISTORY="curl -sf http://history.stellar.org/prd/core-live/core_live_001/{0} -o {1}"
    HOME_DOMAIN="www.stellar.org"
    NAME="sdf_live_1"
    PUBLIC_KEY="GCGB2S2KGYARPVIA37HYZXVRM2YZUEXA6S33ZU5BUDC6THSB62LZSTYH"
    [[VALIDATORS]]
    ADDRESS="core-live-b.stellar.org"
    HISTORY="curl -sf http://history.stellar.org/prd/core-live/core_live_002/{0} -o {1}"
    HOME_DOMAIN="www.stellar.org"
    NAME="sdf_live_2"
    PUBLIC_KEY="GCM6QMP3DLRPTAZW2UZPCPX2LF3SXWXKPMP3GKFZBDSF3QZGV2G5QSTK"
    [[VALIDATORS]]
    ADDRESS="core-live-c.stellar.org"
    HISTORY="curl -sf http://history.stellar.org/prd/core-live/core_live_003/{0} -o {1}"
    HOME_DOMAIN="www.stellar.org"
    NAME="sdf_live_3"
    PUBLIC_KEY="GABMKJM6I25XI4K7U6XWMULOUQIQ27BCTMLS6BYYSOWKTBUXVRJSXHYQ"
    [[VALIDATORS]]
    ADDRESS="stellar-full-validator1.bdnodes.net"
    HISTORY="curl -sf https://stellar-full-history1.bdnodes.net/{0} -o {1}"
    HOME_DOMAIN="stellar.blockdaemon.com"
    NAME="blockdaemon_1"
    PUBLIC_KEY="GAAV2GCVFLNN522ORUYFV33E76VPC22E72S75AQ6MBR5V45Z5DWVPWEU"
    [[VALIDATORS]]
    ADDRESS="stellar-full-validator2.bdnodes.net"
    HISTORY="curl -sf https://stellar-full-history2.bdnodes.net/{0} -o {1}"
    HOME_DOMAIN="stellar.blockdaemon.com"
    NAME="blockdaemon_2"
    PUBLIC_KEY="GAVXB7SBJRYHSG6KSQHY74N7JAFRL4PFVZCNWW2ARI6ZEKNBJSMSKW7C"
    [[VALIDATORS]]
    ADDRESS="stellar-full-validator3.bdnodes.net"
    HISTORY="curl -sf https://stellar-full-history3.bdnodes.net/{0} -o {1}"
    HOME_DOMAIN="stellar.blockdaemon.com"
    NAME="Blockdaemon Validator 3"
    PUBLIC_KEY="GAYXZ4PZ7P6QOX7EBHPIZXNWY4KCOBYWJCA4WKWRKC7XIUS3UJPT6EZ4"
    [[VALIDATORS]]
    ADDRESS="finland.stellar.coinqvest.com"
    HISTORY="curl -sf https://finland.stellar.coinqvest.com/history/{0} -o {1}"
    HOME_DOMAIN="coinqvest.com"
    NAME="coinqvest_fi"
    PUBLIC_KEY="GADLA6BJK6VK33EM2IDQM37L5KGVCY5MSHSHVJA4SCNGNUIEOTCR6J5T"
    [[VALIDATORS]]
    ADDRESS="hongkong.stellar.coinqvest.com"
    HISTORY="curl -sf https://hongkong.stellar.coinqvest.com/history/{0} -o {1}"
    HOME_DOMAIN="coinqvest.com"
    NAME="coinqvest_hk"
    PUBLIC_KEY="GAZ437J46SCFPZEDLVGDMKZPLFO77XJ4QVAURSJVRZK2T5S7XUFHXI2Z"
    [[VALIDATORS]]
    ADDRESS="germany.stellar.coinqvest.com"
    HISTORY="curl -sf https://germany.stellar.coinqvest.com/history/{0} -o {1}"
    HOME_DOMAIN="coinqvest.com"
    NAME="coinqvest_de"
    PUBLIC_KEY="GD6SZQV3WEJUH352NTVLKEV2JM2RH266VPEM7EH5QLLI7ZZAALMLNUVN"
    [[VALIDATORS]]
    ADDRESS="stellar-de-fra.satoshipay.io"
    HISTORY="curl -sf https://stellar-history-de-fra.satoshipay.io/{0} -o {1}"
    HOME_DOMAIN="satoshipay.io"
    NAME="satoshipay_de"
    PUBLIC_KEY="GC5SXLNAM3C4NMGK2PXK4R34B5GNZ47FYQ24ZIBFDFOCU6D4KBN4POAE"
    [[VALIDATORS]]
    ADDRESS="stellar-sg-sin.satoshipay.io"
    HISTORY="curl -sf https://stellar-history-sg-sin.satoshipay.io/{0} -o {1}"
    HOME_DOMAIN="satoshipay.io"
    NAME="satoshipay_sg"
    PUBLIC_KEY="GBJQUIXUO4XSNPAUT6ODLZUJRV2NPXYASKUBY4G5MYP3M47PCVI55MNT"
    [[VALIDATORS]]
    ADDRESS="stellar-us-iowa.satoshipay.io"
    HISTORY="curl -sf https://stellar-history-us-iowa.satoshipay.io/{0} -o {1}"
    HOME_DOMAIN="satoshipay.io"
    NAME="satoshipay_us"
    PUBLIC_KEY="GAK6Z5UVGUVSEK6PEOCAYJISTT5EJBB34PN3NOLEQG2SUKXRVV2F6HZY"
    [[VALIDATORS]]
    ADDRESS="bootes.publicnode.org"
    HISTORY="curl -sf https://bootes-history.publicnode.org/{0} -o {1}"
    HOME_DOMAIN="publicnode.org"
    NAME="bootes"
    PUBLIC_KEY="GCVJ4Z6TI6Z2SOGENSPXDQ2U4RKH3CNQKYUHNSSPYFPNWTLGS6EBH7I2"
    [[VALIDATORS]]
    ADDRESS="hercules.publicnode.org"
    HISTORY="curl -sf https://hercules-history.publicnode.org/{0} -o {1}"
    HOME_DOMAIN="publicnode.org"
    NAME="hercules"
    PUBLIC_KEY="GBLJNN3AVZZPG2FYAYTYQKECNWTQYYUUY2KVFN2OUKZKBULXIXBZ4FCT"
    [[VALIDATORS]]
    ADDRESS="lyra.publicnode.org"
    HISTORY="curl -sf https://lyra-history.publicnode.org/{0} -o {1}"
    HOME_DOMAIN="publicnode.org"
    NAME="lyra"
    PUBLIC_KEY="GCIXVKNFPKWVMKJKVK2V4NK7D4TC6W3BUMXSIJ365QUAXWBRPPJXIR2Z"
    [[VALIDATORS]]
    ADDRESS="v1.stellar.lobstr.co"
    HISTORY="curl -sf https://stellar-archive-1-lobstr.s3.amazonaws.com/{0} -o {1}"
    HOME_DOMAIN="lobstr.co"
    NAME="lobstr_1_eu"
    PUBLIC_KEY="GCFONE23AB7Y6C5YZOMKUKGETPIAJA4QOYLS5VNS4JHBGKRZCPYHDLW7"
    [[VALIDATORS]]
    ADDRESS="v2.stellar.lobstr.co"
    HISTORY="curl -sf https://stellar-archive-2-lobstr.s3.amazonaws.com/{0} -o {1}"
    HOME_DOMAIN="lobstr.co"
    NAME="lobstr_2_eu"
    PUBLIC_KEY="GDXQB3OMMQ6MGG43PWFBZWBFKBBDUZIVSUDAZZTRAWQZKES2CDSE5HKJ"
    [[VALIDATORS]]
    ADDRESS="v3.stellar.lobstr.co"
    HISTORY="curl -sf https://stellar-archive-3-lobstr.s3.amazonaws.com/{0} -o {1}"
    HOME_DOMAIN="lobstr.co"
    NAME="lobstr_3_north_america"
    PUBLIC_KEY="GD5QWEVV4GZZTQP46BRXV5CUMMMLP4JTGFD7FWYJJWRL54CELY6JGQ63"
    [[VALIDATORS]]
    ADDRESS="v4.stellar.lobstr.co"
    HISTORY="curl -sf https://stellar-archive-4-lobstr.s3.amazonaws.com/{0} -o {1}"
    HOME_DOMAIN="lobstr.co"
    NAME="lobstr_4_asia"
    PUBLIC_KEY="GA7TEPCBDQKI7JQLQ34ZURRMK44DVYCIGVXQQWNSWAEQR6KB4FMCBT7J"
    [[VALIDATORS]]
    ADDRESS="v5.stellar.lobstr.co"
    HISTORY="curl -sf https://stellar-archive-5-lobstr.s3.amazonaws.com/{0} -o {1}"
    HOME_DOMAIN="lobstr.co"
    NAME="lobstr_5_au"
    PUBLIC_KEY="GA5STBMV6QDXFDGD62MEHLLHZTPDI77U3PFOD2SELU5RJDHQWBR5NNK7"
---
# Source: horizon/templates/horizon-ingest-cm.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: my-horizon-ingest-env
  namespace: default
  labels:
    app: my-horizon-ingest
    chart: horizon-0.0.7
    release: my-horizon
    heritage: Helm
data:
  ADMIN_PORT: "6000"
  APPLY_MIGRATIONS: "false"
  ALLOW_EMPTY_LEDGER_DATA_RESPONSES: "true"
  CAPTIVE_CORE_CONFIG_APPEND_PATH: "/config/stellar-core.cfg"
  CAPTIVE_CORE_REUSE_STORAGE_PATH: "True"
  CAPTIVE_CORE_REUSE_STORAGE_DIR: "True"
  CAPTIVE_CORE_STORAGE_PATH: "/var/lib/stellar"
  CAPTIVE_CORE_USE_DB: "False"
  CONNECTION_TIMEOUT: "10"
  ENABLE_ACCOUNTS_FOR_SIGNER: "false"
  ENABLE_ASSET_STATS: "true"
  ENABLE_CAPTIVE_CORE_INGESTION: "true"
  ENABLE_EXPERIMENTAL_INGESTION: "false"
  HISTORY_ARCHIVE_URLS: "https://history.stellar.org/prd/core-live/core_live_001,https://history.stellar.org/prd/core-live/core_live_002,https://history.stellar.org/prd/core-live/core_live_003"
  HISTORY_RETENTION_COUNT: "34560"
  INGEST_FAILED_TRANSACTIONS: "true"
  INGEST: "true"
  MAX_DB_CONNECTIONS: "20"
  NETWORK_PASSPHRASE: "Public Global Stellar Network ; September 2015"
  PER_HOUR_RATE_LIMIT: "3600"
  PORT: "8000"
  SSE_UPDATE_FREQUENCY: "4"
  STELLAR_CORE_BINARY_PATH: "/usr/bin/stellar-core"
  STELLAR_CORE_URL: "http://127.0.0.1:11626"
  TMPDIR: "/var/lib/stellar"
---
# Source: horizon/templates/horizon-ingest-sts.yaml
apiVersion: v1
kind: Service
metadata:
  name: my-horizon-ingest-core
  namespace: default
  labels:
    app: my-horizon-ingest-core
    chart: horizon-0.0.7
    release: my-horizon
    heritage: Helm
spec:
  type: ClusterIP
  ports:
    - name: core
      port: 11626
      targetPort: 11626
  selector:
    app: my-horizon-ingest
---
# Source: horizon/templates/horizon-ingest-sts.yaml
apiVersion: v1
kind: Service
metadata:
  name: my-horizon-ingest
  namespace: default
  labels:
    app: my-horizon-ingest
    chart: horizon-0.0.7
    release: my-horizon
    heritage: Helm
spec:
  type: ClusterIP
  ports:
    - name: horizon
      port: 80
      targetPort: 8000
  selector:
    app: my-horizon-ingest
---
# Source: horizon/templates/horizon-ingest-sts.yaml
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: my-horizon-ingest
  namespace: default
  labels:
    app: my-horizon-ingest
    chart: horizon-0.0.7
    release: my-horizon
    heritage: Helm
spec:
  replicas: 1
  serviceName: my-horizon-ingest
  selector:
    matchLabels:
      app: my-horizon-ingest
  template:
    metadata:
      labels:
        app: my-horizon-ingest
        release: my-horizon
        horizon_network: pubnet
    spec:
      containers:
      - name: horizon
        image: "docker.io/stellar/stellar-horizon:2.26.1"
        args:
          - "--apply-migrations"
        imagePullPolicy: Always
        ports:
        - containerPort: 8000
          name: horizon
        - containerPort: 6000
          name: metrics
        - containerPort: 11626
          name: core
        envFrom:
        - secretRef:
            name: 
        - configMapRef:
            name: my-horizon-ingest-env
        readinessProbe:
          httpGet:
            port: 8000
            path: /health
          initialDelaySeconds: 5
          timeoutSeconds: 5
        volumeMounts:
        - mountPath: /config
          name: core-config
      - name: stellar-core-prometheus-exporter
        image: "docker.io/stellar/stellar-core-prometheus-exporter:latest"
        imagePullPolicy: Always
        ports:
        - containerPort: 9473
          name: metrics
      volumes:
      - name: core-config
        configMap:
          name: my-horizon-core

```

File: charts/soroban-rpc/values-pubnet.yaml [hunk 1]
```diff
@@ -48,138 +48,159 @@ sorobanRpc:
...
-        homeDomain: "satoshipay.io"
-        name: "satoshipay_sg"
-        publicKey: "GBJQUIXUO4XSNPAUT6ODLZUJRV2NPXYASKUBY4G5MYP3M47PCVI55MNT"
-      - address: "stellar-us-iowa.satoshipay.io"
-        history: "curl -sf https://stellar-history-us-iowa.satoshipay.io/{0} -o {1}"
-        homeDomain: "satoshipay.io"
-        name: "satoshipay_us"
-        publicKey: "GAK6Z5UVGUVSEK6PEOCAYJISTT5EJBB34PN3NOLEQG2SUKXRVV2F6HZY"
-      - address: "bootes.publicnode.org"
+    validators
```
Comment (sreuland) [inline charts/soroban-rpc/values-pubnet.yaml:65]: yaml typo, s/b `validators:`

retested helm with that change on local copy, output looks good:
```
$ helm template soroban-test ./charts/soroban-rpc --values /Users/sreuland/Downloads/values-pubnet.yaml 
---
# Source: soroban-rpc/templates/soroban-rpc-cm.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: soroban-test-soroban-rpc
  namespace: default
  labels:
    app: soroban-test-soroban-rpc
    chart: soroban-rpc-0.1.1
    release: soroban-test
    heritage: Helm
data:
  soroban-rpc.cfg: |
    CAPTIVE_CORE_CONFIG_PATH="/config/stellar-captive-core.cfg"
    CAPTIVE_CORE_STORAGE_PATH="/var/lib/stellar/captive-core"
    CAPTIVE_CORE_USE_DB=true
    STELLAR_CORE_BINARY_PATH="/usr/bin/stellar-core"
    ENDPOINT="0.0.0.0:8000"
    ADMIN_ENDPOINT="0.0.0.0:6061"
    DB_PATH="/var/lib/stellar/soroban-rpc-db.sqlite"
    STELLAR_CAPTIVE_CORE_HTTP_PORT=11626
    FRIENDBOT_URL="https://friendbot.stellar.org/"
    NETWORK_PASSPHRASE="Public Global Stellar Network ; September 2015"
    HISTORY_ARCHIVE_URLS="https://stellar-history-usw.franklintempleton.com/azuswshf401,https://stellar-archive-1-lobstr.s3.amazonaws.com,https://hercules-history.publicnode.org,https://stellar-full-history1.bdnodes.net,https://history.stellar.org/prd/core-live/core_live_003,https://history.stellar.org/prd/core-live/core_live_002,https://history.stellar.org/prd/core-live/core_live_001"
    CHECKPOINT_FREQUENCY=64
  stellar-captive-core.cfg: |
    DATABASE="sqlite3:///var/lib/stellar/stellar-captive-core.db"
    HTTP_PORT=11626
    PUBLIC_HTTP_PORT=true
    PEER_PORT=11625
    EXPERIMENTAL_BUCKETLIST_DB=true
    EXPERIMENTAL_BUCKETLIST_DB_INDEX_PAGE_SIZE_EXPONENT=12
    ENABLE_SOROBAN_DIAGNOSTIC_EVENTS=true
    NETWORK_PASSPHRASE="Public Global Stellar Network ; September 2015"
    UNSAFE_QUORUM=true
    
    [[HOME_DOMAINS]]
    HOME_DOMAIN="publicnode.org"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="lobstr.co"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="www.franklintempleton.com"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="satoshipay.io"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="whalestack.com"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="www.stellar.org"
    QUALITY="HIGH"
    [[HOME_DOMAINS]]
    HOME_DOMAIN="stellar.blockdaemon.com"
    QUALITY="HIGH"
    [[VALIDATORS]]
    NAME="Boötes"
    HOME_DOMAIN="publicnode.org"
    PUBLIC_KEY=""
    ADDRESS="bootes.publicnode.org:11625"
    HISTORY="curl -sf https://bootes-history.publicnode.org/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Lyra by BP Ventures"
    HOME_DOMAIN="publicnode.org"
    PUBLIC_KEY=""
    ADDRESS="lyra.publicnode.org:11625"
    HISTORY="curl -sf https://lyra-history.publicnode.org/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Hercules by OG Technologies"
    HOME_DOMAIN="publicnode.org"
    PUBLIC_KEY=""
    ADDRESS="hercules.publicnode.org:11625"
    HISTORY="curl -sf https://hercules-history.publicnode.org/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 3 (North America)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v3.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v3.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 1 (Europe)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v1.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v1.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 2 (Europe)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v2.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v2.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 4 (Asia)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v4.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v4.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="LOBSTR 5 (India)"
    HOME_DOMAIN="lobstr.co"
    PUBLIC_KEY=""
    ADDRESS="v5.stellar.lobstr.co:11625"
    HISTORY="curl -sf https://archive.v5.stellar.lobstr.co/{0} -o {1}"
    [[VALIDATORS]]
    NAME="FT SCV 2"
    HOME_DOMAIN="www.franklintempleton.com"
    PUBLIC_KEY=""
    ADDRESS="stellar2.franklintempleton.com:11625"
    HISTORY="curl -sf https://stellar-history-usc.franklintempleton.com/azuscshf401/{0} -o {1}"
    [[VALIDATORS]]
    NAME="FT SCV 3"
    HOME_DOMAIN="www.franklintempleton.com"
    PUBLIC_KEY=""
    ADDRESS="stellar3.franklintempleton.com:11625"
    HISTORY="curl -sf https://stellar-history-ins.franklintempleton.com/azinsshf401/{0} -o {1}"
    [[VALIDATORS]]
    NAME="FT SCV 1"
    HOME_DOMAIN="www.franklintempleton.com"
    PUBLIC_KEY=""
    ADDRESS="stellar1.franklintempleton.com:11625"
    HISTORY="curl -sf https://stellar-history-usw.franklintempleton.com/azuswshf401/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SatoshiPay Frankfurt"
    HOME_DOMAIN="satoshipay.io"
    PUBLIC_KEY=""
    ADDRESS="stellar-de-fra.satoshipay.io:11625"
    HISTORY="curl -sf https://stellar-history-de-fra.satoshipay.io/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SatoshiPay Singapore"
    HOME_DOMAIN="satoshipay.io"
    PUBLIC_KEY=""
    ADDRESS="stellar-sg-sin.satoshipay.io:11625"
    HISTORY="curl -sf https://stellar-history-sg-sin.satoshipay.io/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SatoshiPay Iowa"
    HOME_DOMAIN="satoshipay.io"
    PUBLIC_KEY=""
    ADDRESS="stellar-us-iowa.satoshipay.io:11625"
    HISTORY="curl -sf https://stellar-history-us-iowa.satoshipay.io/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Whalestack (Germany)"
    HOME_DOMAIN="whalestack.com"
    PUBLIC_KEY=""
    ADDRESS="germany.stellar.whalestack.com:11625"
    HISTORY="curl -sf https://germany.stellar.whalestack.com/history/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Whalestack (Hong Kong)"
    HOME_DOMAIN="whalestack.com"
    PUBLIC_KEY=""
    ADDRESS="hongkong.stellar.whalestack.com:11625"
    HISTORY="curl -sf https://hongkong.stellar.whalestack.com/history/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Whalestack (Finland)"
    HOME_DOMAIN="whalestack.com"
    PUBLIC_KEY=""
    ADDRESS="finland.stellar.whalestack.com:11625"
    HISTORY="curl -sf https://finland.stellar.whalestack.com/history/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SDF 2"
    HOME_DOMAIN="www.stellar.org"
    PUBLIC_KEY=""
    ADDRESS="core-live-b.stellar.org:11625"
    HISTORY="curl -sf http://history.stellar.org/prd/core-live/core_live_002/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SDF 1"
    HOME_DOMAIN="www.stellar.org"
    PUBLIC_KEY=""
    ADDRESS="core-live-a.stellar.org:11625"
    HISTORY="curl -sf http://history.stellar.org/prd/core-live/core_live_001/{0} -o {1}"
    [[VALIDATORS]]
    NAME="SDF 3"
    HOME_DOMAIN="www.stellar.org"
    PUBLIC_KEY=""
    ADDRESS="core-live-c.stellar.org:11625"
    HISTORY="curl -sf http://history.stellar.org/prd/core-live/core_live_003/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Blockdaemon Validator 3"
    HOME_DOMAIN="stellar.blockdaemon.com"
    PUBLIC_KEY=""
    ADDRESS="stellar-full-validator3.bdnodes.net:11625"
    HISTORY="curl -sf https://stellar-full-history3.bdnodes.net/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Blockdaemon Validator 2"
    HOME_DOMAIN="stellar.blockdaemon.com"
    PUBLIC_KEY=""
    ADDRESS="stellar-full-validator2.bdnodes.net:11625"
    HISTORY="curl -sf https://stellar-full-history2.bdnodes.net/{0} -o {1}"
    [[VALIDATORS]]
    NAME="Blockdaemon Validator 1"
    HOME_DOMAIN="stellar.blockdaemon.com"
    PUBLIC_KEY=""
    ADDRESS="stellar-full-validator1.bdnodes.net:11625"
    HISTORY="curl -sf https://stellar-full-history1.bdnodes.net/{0} -o {1}"
---
# Source: soroban-rpc/templates/soroban-rpc-ingress.yaml
apiVersion: v1
kind: Service
metadata:
  name: soroban-test-soroban-rpc
  namespace: default
  labels:
    app: soroban-test-soroban-rpc
    chart: soroban-rpc-0.1.1
    release: soroban-test
    heritage: Helm
spec:
  type: ClusterIP
  ports:
    - name: soroban-rpc
      port: 8000
      targetPort: 8000
  selector:
    app: soroban-test-soroban-rpc
---
# Source: soroban-rpc/templates/soroban-rpc-sts.yaml
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: soroban-test-soroban-rpc
  namespace: default
  labels:
    app: soroban-test-soroban-rpc
    chart: soroban-rpc-0.1.1
    release: soroban-test
    heritage: Helm
spec:
  # Currently Soroban RPC doesn't support HA deployments over a single shared data store
  # but, each replica pod in a statefulset will have it's own data store on it's PVC.
  # This statefulset creates two replicas which will be monitored by the readiness probe. 
  # Having two replicas also ensures zero downtime during upgrades as at least one replica is 
  # gauranteed to be running while the other one is upgraded and catching backup to network 
  # after restart
  replicas: 2
  serviceName: soroban-test-soroban-rpc
  selector:
    matchLabels:
      app: soroban-test-soroban-rpc
  template:
    metadata:
      labels:
        app: soroban-test-soroban-rpc
        release: soroban-test
    spec:
      containers:
      - name: soroban-rpc
        image: "docker.io/stellar/soroban-rpc:0.9.3"
        args: ["--config-path", "/config/soroban-rpc.cfg"]
        imagePullPolicy: Always
        volumeMounts:
        - name: config-volume
          mountPath: /config
          readOnly: true
        ports:
        - containerPort: 8000
          name: soroban-rpc
        - containerPort: 6061
          name: admin-port
        readinessProbe:
          exec:
            command:
              - /bin/sh
              - -c
              - |
                curl --location --request POST 'http://127.0.0.1:8000/' \
                  --header 'Content-Type: application/json' \
                  --data-raw '{
                    "jsonrpc": "2.0",
                    "id": 10235,
                    "method": "getHealth"
                  }' | jq -es 'if (. | length) == 0 then null else .[0] end | .result | .status == "healthy" and (.latestLedger - .oldestLedger >= (.ledgerRetentionWindow - 10))' > /dev/null;
          failureThreshold: 2
          periodSeconds: 10
          successThreshold: 2
          timeoutSeconds: 5
      volumes:
      - name: config-volume
        configMap:
          name: soroban-test-soroban-rpc
---
# Source: soroban-rpc/templates/soroban-rpc-ingress.yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: soroban-test-soroban-rpc
  namespace: default
spec:
  tls:
  - secretName: soroban-test-soroban-rpc-cert
    hosts:
    - "example.com"
  rules:
  - host: "example.com"
    http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: soroban-test-soroban-rpc
            port:
              number: 8000

```

File: charts/horizon/templates/_core-config.tpl
Comment (sreuland) [reply charts/horizon/templates/_core-config.tpl:1]: unrelated, but this horizon helm activity reminded me of pending updates to stellar docs to explain how to use horizon helm, but it's been staged for a while - https://github.com/stellar/stellar-docs/pull/216, I probably need to nudge that forward as the content would likely be helpful.

File: charts/soroban-rpc/values-pubnet.yaml (hunk 1 above)
Comment (mollykarcher) [reply charts/soroban-rpc/values-pubnet.yaml:65]: Doh! Thanks, nice catch!

**Review by sreuland — APPROVED** [review]

**Pull request merged**